binding/json_test.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2019 Gin Core Team. All rights reserved.
+// Use of this source code is governed by a MIT style
+// license that can be found in the LICENSE file.
+
+package binding
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
```

Comment:
need the package?

Reply:
Fixed, I've added that package in vendor file.

That package helps us to show an useful test output on error cases. In case when we see assert's message of `err != nil`, the message about `field's value is not equals` says nothing helpful.